.github/workflows/net-ci.yml

```diff
@@ -0,0 +1,35 @@
+name: .NET CI
```

Comment:
Adding this file is also in your other PR, so
a) same comment about the branch name
b) you might get merge issues

Reply:
I left this file by accident! Nice catch! I will remove it.